src/strategy/periodic.rs

```diff
@@ -50,6 +50,17 @@ impl StrategyPeriodic {
         self.schedule.length_minutes()
     }
 
+    /// Return the remaining duration to next window, in human terms.
+    pub(crate) fn human_remaining(&self) -> String {
+        let datetime = chrono::Utc::now();
+        let remaining = self.schedule.remaining_to_datetime(&datetime);
+        match remaining {
+            None => "unavailable".to_string(),
```

Comment:
Minor/optional: should this be e.g. "not found" so that the description reads "next window not found" instead?